app/views/rubygems/_aside.html.erb

```diff
@@ -51,6 +51,13 @@
     </i>
   </h2>
 
+  <h2 class="gem__ruby-version__heading t-list__heading">
+    <%= t('.requires_mfa') %>:
+    <span class="gem__ruby-version">
+      <%= @latest_version.rubygems_mfa_required? ? t(".yes") : t(".no") %>
```

Comment:
When `yes`, this gives the false impression that all previous versions of the gem have required MFA. Could this show more exact information about which version(s) required MFA?

Reply:
Good idea. One solution might be to show the version that introduced MFA like "MFA REQUIRED since 0.0.1". Additionally, version listing might be a good place to show when MFA requirement was introduced (and dropped pontentially).